app/src/main/java/nl/nfi/cellscanner/MainActivity.java

```diff
@@ -191,23 +158,16 @@ public void onClick(DialogInterface dialog, int which) {
     public void startRecording() {
         Context ctx = getApplicationContext();
         if (requestLocationPermission()) {
-            LocationService.start(this);
+            Recorder.startService(this);
             Toast.makeText(ctx, "Location service started", Toast.LENGTH_SHORT).show();
             Log.v(App.TITLE, "Location service started");
         } else {
             Toast.makeText(ctx, "no permission -- try again", Toast.LENGTH_SHORT).show();
```

Comment:
Not part of current PR but
```suggestion
            Toast.makeText(ctx, "App has no permission to use location data. Change your app settings and try again", Toast.LENGTH_SHORT).show();
```